lib/playlist.js

```diff
@@ -84,9 +85,15 @@ module.exports = function (spotify) {
 
   function loadPlaylist(playlist){
     firebaseRef.child("name").set(playlist.name);
-    tracks = playlist.getTracks().map(function(track){
-      return (localFileRegex.test(track.link)) ? null : track.link;
+
+    var availableTracks = _.filter(playlist.getTracks(), function(track) {
+      return track.availability !== 0 && !localFileRegex.test(track.link);
```

Comment:
Since .map will ignore null values you can adjust the current .map-call like this:
```
return (localFileRegex.test(track.link) && track.availability == 0) ? null : track.link;
```
Less code and no need to add lodash.